### PR TITLE
Introduce GetPfIndexByVfPciAddress

### PR DESCRIPTION
**Pull request by bn222**

### CODE REVIEW

File: sriovnet.go [hunk 1]
```diff
@@ -433,6 +433,24 @@ func GetVfIndexByPciAddress(vfPciAddress string) (int, error) {
 	return -1, fmt.Errorf("vf index for %s not found", vfPciAddress)
 }
 
+// gets the PF index that's associated with a VF PCI address (e.g '0000:03:00.4')
+func GetPfIndexByVfPciAddress(vfPciAddress string) (int, error) {
```
Comment (adrianchiris) [inline sriovnet.go:437]: Overall Looks good.

can you add a UT for it ?

**Comment by adrianchiris** [comment]
can you add more details in commit message ?

File: sriovnet.go (hunk 1 above)
Comment (bn222) [reply sriovnet.go:437]: Done. PTAL

Comment (adrianchiris) [reply sriovnet.go:437]: i dont see the UT, did you push the latest changes ?

Comment (bn222) [reply sriovnet.go:437]: Pushed it.

**Comment by wizhaoredhat** [comment]
LGTM

**Review by rollandf — COMMENTED** [review]
LGTM

**Review by rollandf — APPROVED** [review]

**Pull request merged**